diff -uNr FTL-6.7.orig/src/version.c.in FTL-6.7/src/version.c.in
--- FTL-6.7.orig/src/version.c.in	2026-07-06 17:07:11.000000000 -0300
+++ FTL-6.7/src/version.c.in	2026-07-18 01:43:07.663723334 -0300
@@ -12,33 +12,33 @@
 
 const char * __attribute__ ((const)) git_version(void)
 {
-	return "@GIT_VERSION@";
+	return "v6.7";
 }
 
 const char * __attribute__ ((const)) git_date(void)
 {
-	return "@GIT_DATE@";
+	return "20260706";
 }
 const char * __attribute__ ((const)) git_branch(void)
 {
-	return "@GIT_BRANCH@";
+	return "master";
 }
 const char * __attribute__ ((const)) git_tag(void)
 {
-	return "@GIT_TAG@";
+	return "v6.7";
 }
 
 const char * __attribute__ ((const)) git_hash(void)
 {
-	return "@GIT_HASH@";
+	return "fa65a88";
 }
 
 const char * __attribute__ ((const)) ftl_arch(void)
 {
-	return "@FTL_ARCH@";
+	return "x86_64";
 }
 
 const char * __attribute__ ((const)) ftl_cc(void)
 {
-	return "@FTL_CC@";
+	return "gcc 15.3.0 (SUSE Linux)";
 }
