--- a/src/dnsmasq/dnssec.c	2026-02-17 17:20:12.000000000 -0300
+++ b/src/dnsmasq/dnssec.c	2026-03-12 22:44:12.508944825 -0300
@@ -664,7 +664,7 @@
 	    }
 	}
      
-      hash->digest(ctx, hash->digest_size, digest);
+      hash->digest(ctx, digest);
       
       /* namebuff used for workspace above, restore to leave unchanged on exit */
       p = (unsigned char*)(rrset[0]);
@@ -844,7 +844,7 @@
 		 rather then O(keys x DSs) */
 	      hash->update(ctx, (unsigned int)wire_len, (unsigned char *)name);
 	      hash->update(ctx, (unsigned int)rdlen, psave);
-	      hash->digest(ctx, hash->digest_size, digest);
+	      hash->digest(ctx, digest);
 	      
 	      from_wire(name);
 
@@ -1388,13 +1388,13 @@
  
   hash->update(ctx, to_wire(in), (unsigned char *)in);
   hash->update(ctx, salt_len, salt);
-  hash->digest(ctx, hash->digest_size, digest);
+  hash->digest(ctx, digest);
 
   for(i = 0; i < iterations; i++)
     {
       hash->update(ctx, hash->digest_size, digest);
       hash->update(ctx, salt_len, salt);
-      hash->digest(ctx, hash->digest_size, digest);
+      hash->digest(ctx, digest);
     }
    
   from_wire(in);
